dot/rpc/modules/state.go

```diff
@@ -76,6 +76,12 @@ type StateStorageQueryRangeRequest struct {
 	EndBlock   common.Hash `json:"block"`
 }
 
+// StateStorageQueryAtRequest holds json fields
+type StateStorageQueryAtRequest struct {
```

Comment:
I think `StateStorageQueryAtRequest` can be unexported

Reply:
That would apply to its properties as well.

I don't see any variable of type StateStorageQueryAtRequest getting used by scale. Because scale requires structs and properties to be exported.

Reply:
I think they are exported for the encoding/json encoder to work right?

Reply:
Yes, they are exported for encoding/json encoder to work.

Reply:
Yes, we're using Gorilla web toolkit to expose functions through RPC, Gorilla exposes functions with a signature of containing exported structs for request and response.  I tried changing StateStorageQueryAtRequest to un-exported and the RPC function was no longer accessible.